fuzz/marinade/verify-dwarf-addrs.py

```
@@ -0,0 +1,192 @@
+    in_text = [a for a in nonzero if a < text_size]
+
+    if not nonzero:
+        print(f"verify-dwarf-addrs: ERROR no non-zero line addresses in {path}.",
+              file=sys.stderr)
+        return 1
+
+    pct = 100.0 * len(in_text) / len(nonzero)
+    if pct < 50.0:
+        shifted = [a >> 32 for a in nonzero]
+        shifted_ok = sum(1 for a in shifted if 0 < a < text_size)
+        print(f"verify-dwarf-addrs: ERROR coverage would render EMPTY.", file=sys.stderr)
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)